logicalRequirements.md

```diff
@@ -405,6 +405,21 @@ __Additional considerations__
 Please refer to the sections on `leaveWithGModeSetup`, `leaveWithGMode`, and `gModeImmobile` in [the Region documentation](region/region-readme.md) for a more detailed explanation of these objects.
 
 
+#### itemNotCollected object
+An `itemNotCollected` object represents the need to have not yet collected the item at a given node in the same room. For example, such
```

Comment:
I'd like the explanation of what the integer represents in its own sentence so it stands out more.